catlas/nuclearity.py

```diff
@@ -105,12 +102,12 @@ def evaluate_infiniteness(hist, hist_rep):
 
 def get_connectivity_matrix(slab_atoms):
     """
-    Function that gets a connectivity matrix by looking at nearest neighbors.
+    Get connectivity matrix by looking at nearest neighbors.
     Args:
-        slab_atoms: atoms object of the slab
+        slab_atoms (ase.Atoms): a slab object
 
     Returns:
-        overall_connectivity_matrix
+        numpy.ndarray[int8]: an array describing what atoms are connected
```

Comment:
Is this correct? Is the array 0 if no connection, 1 if connection?

Reply:
haha didnt you add that?

Reply:
I think its always a matrix though. If there are no connections it will be a matrix of 0s